src/Altinn.Profile/Controllers/OrganizationsController.cs

```diff
@@ -61,5 +61,34 @@ public async Task<ActionResult<OrganizationResponse>> GetMandatory([FromRoute]st
 
             return Ok(response);
         }
+
+        /// <summary>
+        /// Endpoint looking up the notification addresses for the given organization
+        /// </summary>
+        /// <returns>Returns an overview of the registered notification addresses for the provided organization</returns>
+        [HttpPost("mandatory")]
+        ////[Authorize(Policy = AuthConstants.OrgNotificationAddress_Write)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        public async Task<ActionResult<OrganizationResponse>> CreateNotificationAddress([FromRoute] string organizationNumber, [FromBody] NotificationAddressModel request, CancellationToken cancellationToken)
+        {
+            if (!ModelState.IsValid)
+            {
+                return ValidationProblem(ModelState);
+            }
+
+            if (string.IsNullOrWhiteSpace(organizationNumber))
+            {
+                return BadRequest("Organization number is required");
+            }
+
+            var notificationAddresses = OrganizationRequestMapper.MapNotificationAddress(request);
+
+            var organization = await _notificationAddressService.CreateNotificationAddress(organizationNumber, notificationAddresses, cancellationToken);
+
+            var response = OrganizationResponseMapper.MapResponse(organization);
+
+            return Created($"profile/api/v1/organizations/{organizationNumber}/notificationaddresses/mandatory", response);
```

Comment:
For now the response is an organization with a list of notification-addresses, and thus the location returns the path to the organization. 
A more RESTful approach would be to return a location (and model) to the individual notification address that was created. This would harmonize better with individual DELETE and PUT endpoints